### DISCUSSION TITLE
This Situation is Practically Serious... (IMPORTANT MESSAGE AND NOT A JOKE AT ALL)

### DISCUSSION THREAD

**Discussion by VincentAraneoBack** [SuperTux/supertux#3045] [General]
For you, you think that I am a nuisance and yes, I admit it, with my memes that I send in my discord
I already realize that I am bothering you a little and I apologize.
But I'm not going to talk about that,
but about something that you have to know and it's TOO IMPORTANT.

You will see. You will know of a person named
Rafael Quispe. That was a friend of mine from this community. And before we were friends together with Thiago 4802R (Another friend who I am still friends with),
Rafael helped us in the development of an add-on for Thiago called *Corrupted Forest* It had more than 6 chapters or rather worlds to go through.
Until something not so serious happens but it does harm me and my friend...

Suddenly he started to hate me and Thiago for things we didn't do 
(although I don't know if he has anything to do with my case of stealing assets), since now he started insulting us and then left the Corrupted Forest team. 
Then we canceled the add-on and now we are only in Snow And Forest, 
and now we continue working on it. Then in those moments we got addicted to Minetest, and something happened 
that Rafael Destroyed the Village of Someone Known As TuxAnimados (It doesn't have a name in Spanish, we'll just call it tux). Rafael killed the villagers 
of Tux and then he noticed it and said it was me and Thiago, and they unjustly banned us.
Me and thiago got angry and then we left the case behind. But then something happened that I didn't expect. 
Rafael, for one reason, STEAL THE CORRUPTED FOREST PROJECT FROM US, he grabbed the original file of Corrupted Forest v0.5.0 
and made a port for v0.6.3, he also stole some enemies that I edited for Chapter 2 and he didn't give me credits. 
The levels were stolen from Thiago and were not from Rafael.
Then nothing happened, and now what happened to me 1 or 2 days ago happened.
Rafael Quispe spoke to me via DM on Discord and out of nowhere he told me that he was 
Homosexual/Gay without any sense, I took it as a joke but in a while he 
PASSED ME A PAGE OF VIDEOS +18 (I don't want to say the name for fear that 
something would happen) And then he gave me a POTENTIALLY MALICIOUS link that had a virus. 
I don't know exactly if it is a Trojan or malware, but VirusTotal gave me the 
result that there was a potentially malicious virus. Afterwards I SPAME THE LINK OF VIDEOS +18 AND THE LINK WITH VIRUSES TO THE BEST. 
Then he told us that he was going to ruin our lives (I don't know if it affects you)

Then I blocked him and then I banned him from my server for sending the same +18 link to Komaro. 
So yes. Rafael became totally crazy than me and now he wants to "ruin our lives" and make several projects impossible for us.

Please take this totally seriously and not as a joke, 
this really happened and it is something serious that can harm us all. I understand that now they only 
see me as a nuisance, but with Rafael they don't say anything? We are literally facing one of the most disgusting 
people in the fandom and they don't realize it.

So please and I ask you PLEASE, if you see Rafael in your discord friend request list, DO NOT HESITATE TO REPORT HIM AND THEN BLOCK HIM 
so he will not bother you again (key to clarify as he has multiple accounts 
to be undetectable and to that he keeps doing these disgusting things) If you see that he is not joined to a server that they are in, 
IT MEANS THAT HE IS RAFAEL, especially if they are in a server that has a chat in Spanish, 
since he speaks in that language and is not one of the people who write well what we say

Extra: he is potentially a kind of "Pedophile" (sorry for saying that word) 
because he said he wanted me to see something related to a minor and because of Rafael's thoughts 
I doubt it is just a normal photo, Rafael is 16 years old, the I don't know how old he is, but he must be between 11 and 13 years old.

Thanks For Reading...❤

(here is evidence of the case, unfortunately it is in Spanish but I already gave explanations before, if you have any questions, you can tell me)
![me quiso hackear](https://github.com/user-attachments/assets/d55f1bd7-2fb4-4a15-bea4-173a92a77eba)
![Pruebas1](https://github.com/user-attachments/assets/24f2a19f-54de-4737-97b2-9347c4929f58)
![pruebas2](https://github.com/user-attachments/assets/0c6002a9-e9e0-4dba-82f8-9fcb2c850932)

**Comment by tylerandari13** [comment]
I dont know what to say.

**Comment by Frostwithasideofsalt** [comment]
This feels like a personal issue which shouldn't really be discussed here. at least in my opinion.

**Comment by weluvgoatz** [comment]
Ok

**Comment by Alasdairbugs** [comment]
Hi Vincent,

As this did not occur on the SuperTux discord server or any places we moderate, we cannot verify the validity of your claims here and have no ability to moderate the situation, including giving punishments to any bad actors in this situation.

However, I do suggest you talk your Parents regarding this issue as it is still a matter of **safeguarding**. If it is true what you're claiming, please do talk to a parental figure, as there may be real-world legal consequences for the person you are alleging this against. A Parent will know best how to handle the situation going forward.

As for allegations of plagiarism I am sure these names will be remembered and should any add-ons containing plagiarised content come around, the SuperTux Team may reject it on grounds of plagiarism. Members who plagiarise without reflection on their actions may be blacklisted from their add-on content featuring in-game.
